lxdock/conf/schema.py

```diff
@@ -21,6 +21,7 @@
     'shell': {
         'user': str,
         'home': str,
+        'environment': {Extra: str},
```

Comment:
I think that this rule is a bit too restrictive. For example, during my tests I tried to define environment variables as follows:

```
shell:
  environment:
    LC_ALL: en_US.utf8
    TEST_VAR: 42
```

This fails because the `TEST_VAR` variable is converted to an integer after YML loading, and we are expecting a string. :wink: